app/tests/v1/test_product.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+import json
+from app import create_app
+
+
+class ProductTestCase(unittest.TestCase):
+    """This is the class for product test cases"""
+
+    def setUp(self):
+        self.app = create_app('testing')
+        self.client = self.app.test_client
+
+    def test_to_add_product(self):
+        """Test method to add product"""
+        add_product = self.client().post('/api/v1/product/pencil',
+                                         data=json.dumps(
+                                             dict(category='category',
+                                                  Quantity='Quantity',
+                                                  Description='Description',
+                                                  price='price')),
+                                         content_type='application/json')
+        self.assertEqual(add_product.status_code, 201)
```

Comment:
why are you not testing the content to be expected, as i said tests are not a matter of getting the status codes right